velox/docs/monitoring/stats.rst

```diff
@@ -39,3 +39,17 @@ Memory Arbitration
      - kByte
      - The reclaimed memory bytes of an operator during the memory arbitration.
        This stats only applies for spillable operators.
+
+Task Execution
+------------------
+.. list-table::
+   :widths: 40 10 50
+   :header-rows: 1
+
+   * - Stats Name
+     - Type
+     - Description
+   * - joinTableBuildWallNanos
+     - kNano
```

Comment:
kNano doesn't seem right for docs. I see it is used in the docs above as well, but I feel this needs to be fixed.